sentry-android-core/src/test/java/io/sentry/android/core/AppStartStateTest.kt

```diff
@@ -77,4 +77,16 @@ class AppStartStateTest {
 
         assertEquals(400, sut.appStartInterval)
     }
+
+    @Test
+    fun `getAppStartInterval returns null if more than 60s`() {
+        val sut = AppStartState.getInstance()
+
+        val date = Date()
+        sut.setAppStartTime(100, date)
+        sut.setAppStartEnd(70000)
```

Comment:
`m`: I would test closer to the boundary. 
```suggestion
        sut.setAppStartEnd(60100)
```